packages/fastify-react/plugin/virtual.js

```diff
@@ -6,6 +6,7 @@ import { findExports } from 'mlly'
 const __dirname = dirname(fileURLToPath(import.meta.url))
 
 const virtualRoot = resolve(__dirname, '..', 'virtual')
+const virtualRootTS = resolve(__dirname, '..', 'virtual-ts')
```

Comment:
Could you add some comment in the code explaining why virtual modules are needed at all? I'm having difficulty understanding the reason for their existence.

Reply:
Read this section:

https://fastify-vite.dev/react/project-structure#smart-imports

Also Chapter 5 of HLM explains them.

Reminder: recent changes are still fresh for a review, I'm working on it at the moment.